lib/Docs/RST/RSTCopier.php

```diff
@@ -106,7 +101,7 @@ private function prepareRSTTemplate(string $content, string $sidebar, string $so
 
         // append the source file name to the content so we can parse it back out
         // for use in the build process
```

Comment:
This comment is a remnant if the code you removed in the next line. This can be removed too.